README.md

```diff
@@ -37,7 +37,7 @@ tweaked to pass the parser.
 1. `cd regulations-parser`
 1. `git clone https://github.com/cfpb/fr-notices.git`
 1. `pip install -r requirements.txt`
-1. `echo "LOCAL_XML_PATHS = ['fr-notices']" >> local_settings.py`
+1. `echo "LOCAL_XML_PATHS = ['fr-notices/']" >> local_settings.py`
```

Comment:
It probably does, but not in the quick start section since it is basically a list of copy and paste-able instructions.